tests/with_versions.rs

```diff
@@ -25,7 +25,14 @@ fn msrv_using_linear_method(folder: &str, expected_version: semver::Version) {
     let folder = std::path::Path::new(env!("CARGO_MANIFEST_DIR"))
         .join("features")
         .join(folder);
-    let with_args = vec!["cargo", "msrv", "--path", folder.to_str().unwrap()];
+    let with_args = vec![
+        "cargo",
+        "msrv",
+        "--path",
+        folder.to_str().unwrap(),
+        "--release-source",
+        "rust-dist",
+    ];
```

Comment:
I think it may be better to create a new separate test case for `--release-source` :) here.

Reply:
Thanks for catching this! I think I meant to remove it since it should be covered by the new test `msrv_with_release_source`, do you think I should add another?

Reply:
I don't think that's necessary :).